kalite/i18n/__init__.py

```diff
@@ -351,21 +351,6 @@ def set_default_language(lang_code):
     Settings.set("default_language", lcode_to_ietf(lang_code))
 
 
-def get_langs_with_subtitle(youtube_id):
```

Comment:
:fire:

A monstrously confusing function that ultimately does very little, and was only used in one place.